docker/docker-compose.yml

```diff
@@ -1,13 +1,12 @@
-version: "3.4"
 services:
   integritee-node:
     image: "integritee/integritee-node-dev:1.0.14"
     container_name: integritee-node
     networks:
       - integritee-test-network
     command: --dev --rpc-methods unsafe --ws-external --rpc-external --ws-port 9912
-    logging:
-      driver: none
+    #logging:
+      #driver: local
```

Comment:
The node now creates logs (which is the default when nothing is set, just as for the other services)